src/Host/Client/Impl/Host/IBrokerClient.cs

```diff
@@ -11,6 +11,7 @@ public interface IBrokerClient : IDisposable {
         string Name { get; }
```

Comment:
Let's leave it alone then. It is probably possible to have some sort of comparable indentifications but it may be too much of a change at this point.